### PR TITLE
Metrics API calls and related changes

### PR DESCRIPTION
**Pull request by ujovlado**
- [x] ~~statistiky podla `runId`~~ prehodene do ApiController (detail jobu)
- [x] detail jobu `/queue/job/7451?include=metrics`
- [x] list jobov `/queue/jobs?include=metrics`
- [x] novy index (project, date) + command nan (`syrup:queue:metrics-add-project-and-date-index`)
- [x] zagregovane stats podla dna a komponenty `/queue/metrics/project/1006?dateFrom=2016-08-01&dateTo=2016-08-31`
- [x] API docs (http://docs.syrupqueue.apiary.io/)

### CODE REVIEW

**Comment by Halama** [comment]
Ten api call na statistiky pro joby by byl idealni aby nebyl samostatny ale slo o rozsireni soucasneho pomoci `include` parametru:

## Detail jobu
http://docs.syrupqueue.apiary.io/#reference/jobs/job/view-job-detail

`GET https://syrup.keboola.com/queue/jobs/jobId?include=stats`
a v odpovedi by bylo nove pole `stats`.

## List jobu
http://docs.syrupqueue.apiary.io/#reference/jobs/jobs/view-a-list-of-jobs

`GET https://syrup.keboola.com/queue/jobs?include=stats`

Opet pro kazdy job v seznamu vrati statistiky navic, melo by to by implementovane ale tak at to nedela pro kazdy job dotaz do dynama :)

**Comment by Halama** [comment]
Pro agregovane stats pod dni a komponente pak novy API call.

File: Controller/ApiController.php [hunk 1]
```diff
@@ -96,9 +99,20 @@ public function jobAction($jobId)
         $jobData['params'] = (object) $jobData['params'];
         $jobData['result'] = (object) $jobData['result'];
 
+        if (isset($params['include'])) {
+            $include = explode(',', $params['include']);
+            array_walk($include, function (&$val) {
```
Comment (Halama) [inline Controller/ApiController.php]: tohle s tou referenci je docela divoka konstrukce :) bych to udelal hezky imutabilne a citelne pomoci `array_map`.

Comment (ujovlado) [inline Controller/ApiController.php]: fixed:)

**Pull request merged**